templates/misc/zope_urls_from_db.py

```diff
@@ -0,0 +1,22 @@
+import psycopg2
+
+
+connection_string = (
+    'postgres://{{ archive_db_user }}:{{ archive_db_password }}'
```

Comment:
Probably not a great idea to hardcode the database password in a script. The file is readily accessible to anyone that can user authenticate. Basically, it means a backdoor hack is possible, which could easily be used to compromise the whole machine.